### PR TITLE
Add Glass docs, fix duplicate citations

### PR DESCRIPTION
**Pull request by luisbarrancos**
Sphinx-bibtex objects to same citations from the master bib file across multiple documents.
The only solution to this is to use key (and label) prefixing, which changes the citation string.

### CODE REVIEW

File: docs/source/shaders/material/as_glass.rst [hunk 1]
```diff
@@ -27,50 +27,56 @@ Surface Transmittance
     Transmittance color.
```
Comment (dictoon) [inline docs/source/shaders/material/as_glass.rst]: Let's maybe explain this in one short sentence?

File: docs/source/shaders/material/as_glass.rst [hunk 2]
```diff
@@ -27,50 +27,56 @@ Surface Transmittance
...
+    3. *Student's t* :cite:`a-10.1111:cgf.13137`
 
 *Roughness*:
-    Apparent surface roughness
+    The apparent surface roughness, affecting both the reflection and refraction equally.
+
+*Specular Spread*:
+    Specular highlights spread. This controls the *tails* of the specular highlights, with high values producing a softer looking highlights, and lower values producing sharper looking highlights.
+    
+.. note:: This parameter is valid only for the Student's t distribution only. Higher values approach the GGX distribution, and lower spread values, therefore sharper highlights, approach the Beckmann distribution.
```
Comment (dictoon) [inline docs/source/shaders/material/as_glass.rst]: *t-distribution*

File: docs/source/shaders/material/as_glass.rst [hunk 3]
```diff
@@ -27,50 +27,56 @@ Surface Transmittance
...
-    Surface anisotropy amount
+    Overall intensity of the anisotropy effect, with a value of 0.0 representing isotropic specular highlights.
 
 *Anisotropy Angle*:
-    The anisotropy angle, in [0,1] range, maps to [0,360] degrees.
+    Rotation angle for the anisotropic highlights, with the value in [0,1] range mapping to a rotation from 0 to 360 degrees.
 
 *Anisotropy Vector Map*:
-    A color anisotropy vector tangent field, with the X and Y anisotropy encoded in the R and G channels.
+    Also known as tangent field, encodes the anisotropy directions along X and Y in the Red and Green or Red and Blue channels of the image. Appleseed expects values encoded in the Red and Green channels.
```
Comment (dictoon) [inline docs/source/shaders/material/as_glass.rst]: *appleseed*, even at the beginning of sentences :)

File: docs/source/shaders/material/as_glass.rst [hunk 4]
```diff
@@ -103,10 +109,142 @@ Outputs
...
+   :title:
+
+   Hammered glass with slight green absorption. Specular roughness set to 0.01 with the Beckmann MDF.
+
+.. thumbnail:: /_images/screenshots/glass/glass_std_spread1.png
+   :group: shots_as_glass_group_A
+   :width: 10%
+   :title:
+
+   Sharp glass with long specular highlight tails, creating the appearance of a thin veil over the specular highlights. Using the Student t distribution with a specular roughness of 0.05, specular spread of 0.25, and with a IOR set to 1.64.
```
Comment (dictoon) [inline docs/source/shaders/material/as_glass.rst]: *Student's t-distribution*

**Comment by dictoon** [comment]
Brilliant!!

Two things to fix everywhere in the PR:
* *appleseed* is always spelled in lower case
* It's spelled [Student's t-distribution](https://en.wikipedia.org/wiki/Student%27s_t-distribution)

**Comment by dictoon** [comment]
The history of the t-distribution is interesting, it's actually linked to Guinness beer! https://en.wikipedia.org/wiki/Student%27s_t-distribution#History_and_etymology

File: docs/source/shaders/material/as_glass.rst [hunk 5]
```diff
@@ -168,7 +168,7 @@ Screenshots
    :width: 10%
    :title:
 
-   Sharp glass with long specular highlight tails, creating the appearance of a thin veil over the specular highlights. Using the Student t distribution with a specular roughness of 0.05, specular spread of 0.25, and with a IOR set to 1.64.
+   Sharp glass with long specular highlight tails, creating the appearance of a thin veil over the specular highlights. Using the Student t-distribution with a specular roughness of 0.05, specular spread of 0.25, and with a IOR set to 1.64.
```
Comment (dictoon) [inline docs/source/shaders/material/as_glass.rst]: *Student's* (also below) and we're good to merge ;)

Comment (luisbarrancos) [reply docs/source/shaders/material/as_glass.rst]: I think it was added in the previous commit, but let me recheck. Btw, thanks for the historical reference on the distribution. I think i might need to fix the references to the Student t-distribution on our Standard Surface material as well though.

Comment (luisbarrancos) [reply docs/source/shaders/material/as_glass.rst]: yep, there was a reference there as well, fixed below.

**Pull request merged**